src/main/java/sirius/web/http/UserAgent.java

```diff
@@ -75,6 +79,9 @@ protected void parseUserAgent() {
     }
 
     private boolean isWindowsDevice(String lowerCaseUserAgent) {
+        internetExplorer = lowerCaseUserAgent.contains(INTERNET_EXPLORER);
```

Comment:
this only detects IE 11 - is this intended?

Reply:
we could discuss supporting older IE versions as well but that should be a separat discussion i think